docs/kustomization.yaml

```diff
@@ -102,6 +102,24 @@ secretGenerator:
   envCommand: printf \"DB_USERNAME=admin\nDB_PASSWORD=somepw\"
   type: Opaque
 
+# generatorOptions modify behavior of all ConfigMap and Secret generators
+generatorOptions:
+  # labels to add to all generated resources
+  labels:
+    kustomize.generated.resources: somevalue
+  # annotations to add to all generated resources
+  annotations:
+    kustomize.generated.resource: somevalue
+  # timeoutSeconds specifies the timeout for commands
+  timeoutSeconds: 30
+  # shell and arguments to use as a context for commands used in resource
+  # generation. Default at time of writing: ["sh", "-c"]
+  shell: ["sh", "-c"]
+  # disableHash is true disables the default behavior of adding a
+  # suffix to the names of generated resources that is a hash of
+  # the resource contents.
+  disableHash: true
```

Comment:
we should have called this `disableNameSuffixHash`

There's no advantage to abbreviating it and losing a sense of what it actually does.

Perhaps a follow up PR, before people start using this.

Reply:
I think we should update the field name before this doc goes in.

Reply:
done (#525)

Reply:
Thank you 👍